docs/getting_started.md

```diff
@@ -57,6 +57,8 @@ For convenience, it is possible to build the required boost and eigen3 dependenc
 Other options are also
 available and are discussed in the [Building Acts](#building-acts) section.
 
+The prerequisites for profiling the ACTS project with gperftools are detailed in docs/howto/profiling.md
```

Comment:
```suggestion
The prerequisites for profiling the ACTS project with gperftools are detailed [here](howto/profiling.md).
```
but I don't know if that works when converting to readthedocs.

Reply:
I hope not: `getting_started.md` is Markdown after all. But I don't know what defines if something gets added to readthedocs. Maybe @robertlangenberg can advise.